packages/backend/.eslintrc.js

```diff
@@ -52,9 +52,9 @@ module.exports = {
                 'src/projectAdapters/**/*.ts',
             ], 
```

Comment:
it would be nice if we could fix these folders/files and make them error again. But for now, I'm disabling the warning, hopefully that will help with performance.